modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/util/CommerceAccountRoleHelperImpl.java

```diff
@@ -174,6 +197,30 @@ private void _setRolePermissions(Role role, ServiceContext serviceContext)
 					"VIEW_COMMERCE_ORDERS", "VIEW_OPEN_COMMERCE_ORDERS"
 				});
 		}
+		else if (name.equals(
+					AccountRoleConstants.REQUIRED_ROLE_NAME_ACCOUNT_MANAGER)) {
+
+			groupResourceActionIds.put(
+				AccountEntry.class.getName(),
+				new String[] {
+					ActionKeys.UPDATE, ActionKeys.MANAGE_USERS, ActionKeys.VIEW,
+					AccountActionKeys.MANAGE_ADDRESSES,
+					AccountActionKeys.VIEW_ADDRESSES,
+					AccountActionKeys.VIEW_ACCOUNT_ROLES,
+					AccountActionKeys.VIEW_ORGANIZATIONS,
+					AccountActionKeys.VIEW_USERS,
+					CommerceAccountActionKeys.MANAGE_CHANNEL_DEFAULTS,
```

Comment:
in the story it's written that these permission are meant for the administrator I can't find any mention to the account manager

Reply:
I add the new permissions to all the roles that at the moment have the AccountEntry UPDATE permission (so they viewing this tab) otherwise it's a breaking change because someone stops to see/do what he is seeing/doing until now

Reply:
please reach out to Bence and David to better define the _Administrator_ in the story description

Reply:
I think how Stefano managed this is ok. By default only to those that cal already see that page, all others can give or remove the permission later

Reply:
Bence confirms this behavior

> Yes I think it should be checked already after release, because probably not every Customer wants to separate this permission, and it would be annoying for them to provide their Acc Managers and Acc Administrators this permissions every time.